pkgs/applications/science/biology/kent/default.nix

```
@@ -43,22 +43,23 @@ stdenv.mkDerivation rec {
     export CFLAGS="-fPIC"
     export MYSQLINC=$(mysql_config --include | sed -e 's/^-I//g')
     export MYSQLLIBS=$(mysql_config --libs)
-    export DESTBINDIR=$NIX_BUILD_TOP/bin
-    export HOME=$NIX_BUILD_TOP
+    export HOME=$(pwd)
```

```suggestion
    export HOME=$TMPDIR
```
otherwise we might litter the current directory with random files which might end up in $out